README.md

```diff
@@ -153,8 +156,8 @@ jobs:
     # Query is the SQL query that is run unalterted on the each of the connections
     # for this job
     query:  |
-            SELECT datname::text, usename::text, COUNT(*)::float AS count
-            FROM pg_stat_activity GROUP BY datname, usename;
+            SELECT created_at, datname::text, usename::text, COUNT(*)::float AS count
```

Comment:
It doesn't look like this is an existing column name on `pg_stat_activity` according to: https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW

Maybe something like `select count(*), now() as created_at from pg_stat_activity;` would make sense so the timestamp of when the query was run is exported?

Reply:
I figured as much, just didn't want to break this example in case people want to use this metric by default. Thanks for the change. Looks good now! 👍